[Bug]: CI failure collections

### Your current environment

A few CI failure examples:

Example 1: 
http://buildkite.com/vllm/vllm-omni/builds/2261/steps/canvas?sid=019c0421-4c32-4b79-a9ea-f4fa346972aa

```
@pytest.mark.skipif(is_rocm(), reason="Test skipped on AMD environment due to known output issues")
--
@pytest.mark.parametrize("omni_server", test_params, indirect=True)
def test_text_to_text_audio_001(client: openai.OpenAI, omni_server) -> None:
"""
Test text input processing and text/audio output generation via OpenAI API.
Deploy Setting: default yaml
Input Modal: text
Output Modal: text + audio
Datasets: few requests
"""
 
num_concurrent_requests = get_max_batch_size()
messages = dummy_messages_from_mix_data(system_prompt=get_system_prompt(), content_text=get_prompt())
 
e2e_list = list()
with concurrent.futures.ThreadPoolExecutor(max_workers=num_concurrent_requests) as executor:
# Submit multiple completion requests concurrently
futures = [
executor.submit(client.chat.completions.create, model=omni_server.model, messages=messages)
for _ in range(num_concurrent_requests)
]
start_time = time.perf_counter()
# Wait for all requests to complete and collect results
chat_completions = list()
for future in concurrent.futures.as_completed(futures):
chat_completions.append(future.result())
# Verify E2E
current_e2e = time.perf_counter() - start_time
print(f"the request e2e is: {current_e2e}")
# TODO: Verify the E2E latency after confirmation baseline.
e2e_list.append(current_e2e)
 
print(f"the avg e2e is: {sum(e2e_list) / len(e2e_list)}")
# Verify all completions succeeded
assert len(chat_completions) == num_concurrent_requests, "Not all requests succeeded."
for chat_completion in chat_completions:
# Verify audio output success
audio_data = None
text_content = None
for choice in chat_completion.choices:
if choice.message.audio is not None:
audio_message = choice.message
audio_data = audio_message.audio.data
assert audio_message.audio.expires_at > time.time(), "The generated audio has expired."
 
if choice.message.content is not None:
# Verify text output success
text_content = choice.message.content
assert "beijing" in text_content.lower(), "The output do not contain keywords."
 
# Verify text output same as audio output
audio_content = convert_audio_to_text(audio_data)
print(f"text content is: {text_content}")
print(f"audio content is: {audio_content}")
similarity = cosine_similarity_text(audio_content.lower(), text_content.lower())
print(f"similarity is: {similarity}")
>           assert similarity > 0.9, "The audio content is not same as the text"
E           AssertionError: The audio content is not same as the text
E           assert 0.8814825547070535 > 0.9

```



Example 2: 
https://buildkite.com/vllm/vllm-omni/builds/2268/steps/canvas?sid=019c0479-2b60-4259-9cf9-980c2777d9dc

```
FAILED tests/e2e/online_serving/test_qwen3_omni.py::test_text_to_text_audio_001[omni_server0] - AssertionError: The audio content is not same as the text
--
assert 0.8934271123648291 > 0.9
FAILED tests/e2e/online_serving/test_qwen3_omni.py::test_mix_to_text_audio_001[omni_server1] - AssertionError: The audio content is not same as the text
assert 0.49502411434005467 > 0.9

```


### Your code version

from CI

### 🐛 Describe the bug

CI failure due to acc threshold 0.9

### Before submitting a new issue...

- [x] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://vllm-omni.readthedocs.io), which can answer lots of frequently asked questions.

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -920,7 +920,7 @@ class OmniServer:
         )
 
         # Wait for server to be ready
-        max_wait = 600  # 10 minutes
+        max_wait = 1200  # 20 minutes
         start_time = time.time()
         while time.time() - start_time < max_wait:
             try:

